src/qtiCommonRenderer/renderers/Figure.js

```diff
@@ -33,7 +34,7 @@ export default {
         }
 
         let showFigure = false;
-        if (data.attributes.class && ['wrap-left', 'wrap-right'].includes(data.attributes.class)) {
+        if (data.attributes.class && [FLOAT_LEFT_CLASS, FLOAT_RIGHT_CLASS, CENTER_CLASS].includes(data.attributes.class)) {
```

Comment:
1) TODO:
`tao-centered` doesn't work for the image inside _ChoiceInteraction Choice._
Because for some reason `figure` isn't used there, but `wrap-left/wrap-right` is set on `img` itself. 
In that case we should also set `display: block` style.

2) Ignore for now:
`tao-centered` doesn't work inside TextReader PCI content. But `wrap-left/wrap-right` don't either. Image can't be centered by using `<p>` with text-align: center

Reply:
Setting `display: block;` shouldn't hurt in any case. I'll add it.